server/src/main.js

```diff
@@ -81,6 +81,29 @@ app.use(
   }),
 );
 
+// Quick and simple health check endpoint that returns uptime and DB connector health
+app.use('/healthcheck', (_, res) => {
+  let status = 500;
+  const response = {};
+  response.uptime = Math.round(process.uptime());
+  try {
+    db
+      .authenticate()
+      .then(() => {
+        status = 200;
+        response.dbConnector = 'healthy';
+        res.status(status).json(response);
```

Comment:
i think you can probably just omit the status part of this - just let express return what it thinks is sane.

Reply:
run the risk of a unhealthy result returning a code 200 if you do that.

Reply:
(cause we would want a 500 or something if there is a problem)